rotkehlchen/globaldb/utils.py

```diff
@@ -7,7 +7,7 @@
 # Whenever you upgrade the global DB make sure to:
 # 1. Go to assets repo and tweak the min/max schema of the updates
 # 2. Tweak ASSETS_FILE_IMPORT_ACCEPTED_GLOBALDB_VERSIONS
-GLOBAL_DB_VERSION = 4
+GLOBAL_DB_VERSION = 5
```

Comment:
Make sure that this is also still fine for all assets in the update. Seems it is?
https://github.com/rotki/assets/blob/master/updates/info.json

But if we do another upgrade we will need to move the max_assets version for the latest updates.

Reply:
> https://github.com/rotki/assets/blob/master/updates/info.json

Yeah, it is